src/js/step.js

```diff
@@ -247,16 +247,20 @@ export class Step extends Evented {
    * Triggers `destroy` event
    */
   destroy() {
-    if (isElement(this.el) && this.el.parentNode) {
-      this.el.parentNode.removeChild(this.el);
-      delete this.el;
-    }
-
     if (this.tooltip) {
       this.tooltip.destroy();
       this.tooltip = null;
     }
 
+    if (this.el && this.el instanceof HTMLElement) {
+      this.el.remove();
```

Comment:
We need to set this back to the old code. `remove` does not work in IE.

Reply:
Updated